core/gui/src/app/workspace/component/code-editor-dialog/code-editor.component.ts

```diff
@@ -155,7 +166,7 @@ export class CodeEditorComponent implements AfterViewInit, SafeStyle, OnDestroy
       );
     }
     this.editor = editor;
-    this.connectLanguageServer();
+    // this.connectLanguageServer();
```

Comment:
Add if statement to only connect language server for python, and not connect for R